src/client/AIAssist.ts

```diff
@@ -29,49 +29,24 @@ export const AIAssist = ({
     {
       key: 'control-m',
       run: (view: EditorView) => {
-        startAIAssist(view, shareDBDoc, fileId);
-        // fetch(aiAssistEndpoint, {
-        //   method: 'POST',
-        //   headers: {
-        //     'Content-Type': 'application/json',
-        //   },
-        //   body: JSON.stringify({
-        //     // Pass additional options to the AI Assist endpoint.
-        //     ...aiAssistOptions,
-
-        //     text: textToSend,
-        //     fileId,
-        //     cursorLocation: view.state.selection.main.to,
-        //   }),
-        // });
+        if (
+          shareDBDoc.data.aiStreams == null ||
```

Comment:
```suggestion
          shareDBDoc.data.aiStreams === null ||
```

Best practice to prefer `===` over `==` as it is more precise.